riscv/insns/vfncvt_f_f_w.h

```diff
@@ -1,9 +1,7 @@
-// vfncvt.f.f.v vd, vs2, vm
+// vfncvt.f.f.w vd, vs2, vm
```

Comment:
The comment changes can be a separate commit.

Reply:
OK. I have updated it.